src/ruleChecker.ts

```diff
@@ -148,6 +148,12 @@ export default class RuleChecker {
       const hash = createHash('sha256');
       hash.update(findingEvent.hash);
       hash.update(checkInstance.ruleId);
+      if (message) {
+        hash.update(message);
+      }
+      if (groupMessage) {
+        hash.update(groupMessage);
+      }
```

Comment:
This will break our grouping of unique finding occurrences. Here are a few examples of how:
- https://github.com/applandinc/scanner/blob/main/src/rules/nPlusOneQuery.ts#L43
- https://github.com/applandinc/scanner/blob/main/src/rules/slowHttpServerRequest.ts#L12
- https://github.com/applandinc/scanner/blob/main/src/rules/secretInLog.ts#L44